client/mass/test/violin.integration.spec.js

```diff
@@ -673,24 +673,30 @@ tape('test uncomputable categories legend', function(test) {
 		violin.on('postRender.test', null)
 
 		await testUncomputableCategories(violin, legendDiv)
-
+		if (test._ok) violin.Inner.app.destroy()
 		test.end()
 	}
 
 	async function testUncomputableCategories(violin, legendDiv) {
-		const categories = await detectGte({ elem: legendDiv.node(), selector: '.legend-row', count: 9 })
 		const keys = Object.keys(violin.Inner.data.uncomputableValueObj)
-		const category1 = categories[8].innerText.split(',')[0] + ',' + categories[8].innerText.split(',')[1]
-		const category2 = categories[9].innerText.split(',')[0]
+		const categories = await detectGte({ elem: legendDiv.node(), selector: '.legend-row', count: 9 })
+		const uncomputableLegend = categories.filter(c => keys.find(k => c.__data__.text.startsWith(k)))
```

Comment:
@siosonel What about iterating through uncomputableLegend to make the "Uncomputable category..." tests? Then it's only one `test.equal` to write. What do you think?

Reply:
@creilly8 Yes, the tests can be optimized that way